builtin/logical/transit/path_cache_config.go

```diff
@@ -60,8 +60,11 @@ func (b *backend) pathCacheConfigWrite(ctx context.Context, req *logical.Request
 		return nil, err
 	}
 
-	resp := &logical.Response{
-		Warnings: []string{"cache configurations will be applied when this backend is restarted"},
+	resp := &logical.Response{}
+
+	b.lm, err = keysutil.NewLockManager(true, cacheSize)
```

Comment:
I remember somewhere Scott pointing out that we shouldn't be replacing the locks manager and thereby losing track of any existing locks.  Instead, we should be replacing the LockManager `cache` while maintaining the `keyLocks` member.  Not sure what happened to that comment, but anyway, he's right.

Reply:
Yes, that was in a slack thread. I will address that in my next commit. Thanks :)